Add 10.11 release notes (#110)
mondoohq/docs: releases/2024-04-09-mondoo-10.11-is-out.md

---
slug: mondoo-10.11-is-out/
title: Mondoo 10.11 is out!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
tags: [release, mondoo]
---

## 🥳 Mondoo 10.11 is out! This release includes Azure ACR scanning, expanded OS query packs, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

> ### Azure ACR Scanning

Reviewer: ```suggestion
### Azure ACR scanning
```

Mondoo now supports scanning [Azure Container Registries (ACR)](https://azure.microsoft.com/en-us/products/container-registry) that require authentication using credentials stored after running the `az acr login` command.

To login and scan a complete registry run:

```bash
az acr login -n my_registry
cnspec scan container registry my_registry.azurecr.io
```

## 🧹 IMPROVEMENTS

> ### Collect logged in users in query packs

Reviewer: ```suggestion
### Collect logged-in users in query packs
```

> Windows, Linux, and macOS query backs now collect the currently logged in users so you can understand active users on endpoints or remote connections to servers.

Reviewer 2: nit: backs -> packs

> Windows, Linux, and macOS query packs now collect the currently logged in users so you can understand active users on endpoints or remote connections to servers.

Reviewer: ```suggestion
Windows, Linux, and macOS query packs now collect the currently logged-in users so you can understand active users on endpoints or remote connections to servers.
```

## 🐛 BUG FIXES AND UPDATES

- Fix a failure running the `users.all(sshkeys == empty)` query.
- Don't panic when the scan play is set to 0.
- Ignore deactivated users in the Slack policy's 2FA check. Thanks for this fix [@jaybrueder](https://github.com/jaybrueder/)

> - Improve the AWS IAM users multi-factor authentication (MFA) enabled check to only check users with a password set.

Reviewer: ```suggestion
- Improve the AWS IAM user "MFA enabled" check to only check users with a set password.
```

- Fix the discovery of gcp organizations and folders.
- Improve the `scan gcp` help output.

> - Improve failure output when a CLI command could not be parsed in the gcp provider.

Reviewer: ```suggestion
- Improve failure output when a CLI command can't be parsed in the GCP provider.
```
